operator/cmd/root.go

```diff
@@ -516,9 +515,12 @@ func runOperator(log *slog.Logger, lc *LeaderLifecycle, clientset k8sClient.Clie
 				}
 			},
 			OnStoppedLeading: func() {
-				log.Info("Leader election lost", logfields.OperatorID, operatorID)
-				// Cleanup everything here, and exit.
-				shutdowner.Shutdown(hive.ShutdownWithError(errors.New("Leader election lost")))
+				// It's imperative that we stop immediately here. Concurrent execution threads
+				// acting on shared state in k8s/kvstore do so under the assumption that this
+				// operator leads the deployment, which is no longer true once we hit this callback.
+				// If we linger longer than the grace period between the RenewalDeadline and then
```

Comment:
nit:


```suggestion
				// If we linger longer than the grace period between the RenewDeadline and the
```

The `LeaderElectionConfig` field is named `RenewDeadline` so probably good to be consistent. Also `s/then/the`, I think.

Reply:
thx